ui/app/templates/components/drain-popover.hbs

```diff
@@ -0,0 +1,85 @@
+{{#popover-menu
+  data-test-drain-popover
+  label=(if client.isDraining "Update Drain" "Drain")
+  triggerClass=(concat "is-small " (if drain.isRunning "is-loading")) as |m|}}
+  <form data-test-drain-popover-form onsubmit={{action (queue (action preventDefault) (perform drain m.actions.close))}} class="form is-small">
+    <h4 class="group-heading">Drain Options</h4>
+    <div class="field">
+      <label class="label is-interactive">
+        {{#toggle
+          data-test-drain-deadline-toggle
+          isActive=deadlineEnabled
+          onToggle=(action (mut deadlineEnabled) value="target.checked")}}
+          Deadline
+        {{/toggle}}
+        <span class="tooltip multiline" aria-label="The amount of time a drain must complete within.">
+          {{x-icon "info-circle-outline" class="is-faded"}}
+        </span>
+      </label>
+    </div>
+    {{#if deadlineEnabled}}
+      <div class="field is-sub-field">
+        {{#power-select
+          data-test-drain-deadline-option-select
+          tagName="div"
+          options=durationQuickOptions
+          selected=selectedDurationQuickOption
+          onChange=(action (mut selectedDurationQuickOption)) as |opt|}}
+          {{opt.label}}
+        {{/power-select}}
+      </div>
+      {{#if durationIsCustom}}
+        <div class="field is-sub-field">
+          <label class="label">Deadline</label>
+          <input
+            data-test-drain-custom-deadline
+            type="text"
+            class="input {{if parseError "is-danger"}}"
+            placeholder="1h30m"
+            oninput={{action (queue
+              (action (mut parseError) '')
+              (action (mut customDuration) value="target.value"))}} />
+          {{#if parseError}}
+            <em class="help is-danger">{{parseError}}</em>
+          {{/if}}
+        </div>
+      {{/if}}
+    {{/if}}
+    <div class="field">
+      <label class="label is-interactive">
+        {{#toggle
+          data-test-force-drain-toggle
+          isActive=forceDrain
+          onToggle=(action (mut forceDrain) value="target.checked")}}
+          Force Drain
+        {{/toggle}}
+        <span class="tooltip multiline" aria-label="Immediately remove allocations from the client.">
+          {{x-icon "info-circle-outline" class="is-faded"}}
+        </span>
+      </label>
+    </div>
+    <div class="field">
+      <label class="label is-interactive">
+        {{#toggle
+          data-test-system-jobs-toggle
+          isActive=drainSystemJobs
+          onToggle=(action (mut drainSystemJobs) value="target.checked")}}
+          Drain System Jobs
+        {{/toggle}}
+        <span class="tooltip multiline" aria-label="Stop allocations for system jobs.">
+          {{x-icon "info-circle-outline" class="is-faded"}}
```

Comment:
This isn’t quite as necessary as it seemed [in this PR (see GIF)](https://github.com/hashicorp/nomad/pull/5944#issue-295906076) but I wonder if something like the `is-right-aligned` thing would be useful here to address the slight truncation I see:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/43280/70472610-0421e180-1a95-11ea-9451-d3942d2d69e5.png">

Reply:
Yeah, I think that makes sense here.